x/dualstaking/types/delegate.go

```diff
@@ -0,0 +1,48 @@
+package types
+
+import (
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/lavanet/lava/utils/slices"
+)
+
+func NewDelegation(delegator, provider, chainID string) Delegation {
+	return Delegation{
+		Delegator: delegator,
+		Provider:  provider,
+		ChainID:   chainID,
```

Comment:
the coin should be init here with the denom of "ulava"

Reply:
fixed 96a898ea